src/main/java/org/jitsi/jicofo/VersionBroadcaster.java

```diff
@@ -171,18 +172,16 @@ public void handleEvent(Event event)
                     + "(" + jicofoVersion.toString() + ","
                     + System.getProperty("os.name") + ")");
 
-        // Videobridge
-        // It is not be reported for FOCUS_JOINED_ROOM_TOPIC
-        Jid bridgeJid
-                = (Jid)event.getProperty(EventFactory.BRIDGE_JID_KEY);
-        Version jvbVersion
-            = bridgeJid == null
-                ? null : meetServices.getBridgeVersion(bridgeJid);
-        if (jvbVersion != null)
+        String jvbVersions = conference.getBridges().stream()
+            .map(b -> b.getVersion())
+            .filter(Objects::nonNull)
+            .collect(Collectors.joining(", "));
```

Comment:
I think we should treat it as a set (remove duplicates). Otherwise we'll see it change every time we add/remove a bridge to the conference.